From 59f74139c0236cb31831ec51ccfd04fe4b9e7fa0 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 16 May 2022 12:16:13 +0300
Subject: [PATCH 14/30] arm: s32cc: Remove binman node

Remove then binman node and support as the DTB is expected to be
deployed by the ATF.

Issue: ALB-8797
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/dts/s32cc.dtsi   | 10 ----------
 arch/arm/mach-s32/Kconfig |  1 -
 2 files changed, 11 deletions(-)

diff --git a/arch/arm/dts/s32cc.dtsi b/arch/arm/dts/s32cc.dtsi
index 77590d9569..05044183d8 100644
--- a/arch/arm/dts/s32cc.dtsi
+++ b/arch/arm/dts/s32cc.dtsi
@@ -533,15 +533,5 @@
 			interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
 		};
 	};
-
-	binman {
-		filename = "u-boot-s32.bin";
-		u-boot-dtb {
-		};
-		u-boot-nodtb {
-			offset = <CONFIG_S32CC_MAX_DTB_SIZE>;
-		};
-	};
-
 };
 
diff --git a/arch/arm/mach-s32/Kconfig b/arch/arm/mach-s32/Kconfig
index abfdd0844b..e91cec2f11 100644
--- a/arch/arm/mach-s32/Kconfig
+++ b/arch/arm/mach-s32/Kconfig
@@ -35,7 +35,6 @@ config NXP_S32CC
 	imply SYS_I2C_MXC
 	imply TIMER
 	select ARM_SMCCC
-	select BINMAN
 	select CLK
 	select CLK_SCMI
 	select CPU
-- 
2.17.1

